### ISSUE TITLE
Arch Linux package dependancy

### ISSUE THREAD

**Issue by finex** [KaizIqbal/Bibata_Cursor#22]
Hi, the Arch Linux package needs `gnome-themes-standard` to be installed otherwise you will get the following error:  

```
Gtk-WARNING **: Unable to locate theme engine in module_path: "adwaita"
```

**Comment by KaizIqbal** [comment]
You are using AUR  for Installations?

**Comment by finex** [comment]
Yes, I've used AUR. You've wrote `gnome-themes-standard` on the readme but it is missing from the PKGBUILD. I thought you're maintaining the Arch package too, if not I will report on AUR :-)

**Comment by KaizIqbal** [comment]
It Ok Thanks for Report :+1:  
AUR is maintained by @shatur.you contact him for Update PKGBUILD.
